Dockerfile

```diff
@@ -1,4 +1,28 @@
+FROM mcr.microsoft.com/dotnet/sdk:3.1 as builder
+
+ARG GITHUBSHA OPENWEATHERMAP_APIKEY_RAW 
```

Comment:
This won't work, `ARG` only takes a single argument (see https://docs.docker.com/engine/reference/builder/#arg). This needs to be split into separate lines

Additionally, `OPENWEATHER_APIKEY_RAW` should use an empty value `""` as a default. The api key is not needed for this application to work, I'll make sure (in another PR) that this is set in the github pipeline.